src/objects/zcl_abapgit_oo_base.clas.abap

```diff
@@ -168,15 +71,7 @@ CLASS ZCL_ABAPGIT_OO_BASE IMPLEMENTATION.
 
 
   METHOD zif_abapgit_oo_object_fnc~deserialize_source.
-    TRY.
-        deserialize_abap_source_new(
-          is_clskey = is_key
-          it_source = it_source ).
-      CATCH cx_sy_dyn_call_error.
-        deserialize_abap_source_old(
-          is_clskey = is_key
-          it_source = it_source ).
-    ENDTRY.
+    ASSERT 0 = 1. "Subclass responsibility
```

Comment:
why is this moved?

Reply:
Not related to the issue. This code is called only for interfaces, so I think it should be moved at least to subclass ZCL_ABAPGIT_OO_INTERFACE. See my comment in above review.